File: 4 - JAVA/1 - Estrutura-sequencial-digita-segundos-da-horas-minutos-segundos.java
import java.util.Locale;
import java.util.Scanner;

public class Main {
	
	public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int tempo, horas, minutos, segundos, resto;
		
		System.out.print("Digite a duracao em segundos: ");
		tempo = sc.nextInt(); 
			    		
		horas = tempo / 3600;
		resto = tempo % 3600;
		minutos = resto / 60;
		segundos = resto % 60;
		
		System.out.println(horas + ":" + minutos + ":" + segundos);
		
		sc.close();

	}
}

File: 3 - Linguagem C++/1 - Estrutura-sequencial-digita-segundos-da-horas-minutos-segundos.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main()
{
    int seg, segundos, minutos, horas, resto;

    cout << "Digite a duracao em segundos: ";
    cin >> seg;

    horas = seg / 3600;
    resto = seg % 3600;
    minutos = resto / 60;
    segundos = resto % 60;

    cout << "Horas:minutos:segundos = " << horas << ":" <<
    minutos << ":" << segundos << endl;

    return 0;
}
